### PR TITLE
Support AR types serialization

### PR DESCRIPTION
**Pull request by lest**

### CODE REVIEW

File: lib/enumerize/activerecord.rb [hunk 1]
```diff
@@ -59,5 +65,19 @@ def update_all(updates)
         super(updates)
       end
     end
+
+    class Type < ActiveRecord::Type::Value
```
Comment (tkawa) [inline lib/enumerize/activerecord.rb]: This hard-coded `ActiveRecord::Type::Value` probably causes a break 🐛  in Rails 4 (which doesn't have a serialize method). For instance, `ActiveRecord::Type::Integer#type_cast` won't be used in an integer column.

Comment (lest) [inline lib/enumerize/activerecord.rb]: @tkawa Sorry, I'm not sure that I understand how to reproduce the bug you are describing. Can you please push a test application reproducing it?

Comment (tkawa) [inline lib/enumerize/activerecord.rb]: @lest I reproduce it: https://github.com/brainspec/enumerize/issues/272#issuecomment-294330168

Comment (lest) [inline lib/enumerize/activerecord.rb]: It is fixed in the master branch. Thanks for your help!

**Pull request merged**